website/src/main.css

```diff
@@ -0,0 +1,10 @@
+@import "../css/root.css";
+@import "../css/fonts.css";
+@import "../css/navbar.css";
+@import "../css/header.css";
+@import "../css/sidebar.css";
+@import "../css/markdown.css";
+@import "../css/badges.css";
+@import "../css/menu.css";
+@import "../css/buttons.css";
+@import "../css/homepage.css";
```

Comment:
Placing the CSS files in a directory outside `src` allows us to prepare for a package approach. More on this in a second PR.